cuda_core/examples/strided_memory_view.py

```diff
@@ -73,8 +75,18 @@
         source_extension=".cpp",
         extra_compile_args=["-std=c++11"],
     )
-    cpu_prog.compile()
-    cpu_func = getattr(importlib.import_module("_cpu_obj.lib"), func_name)
+    temp_dir = tempfile.mkdtemp()
+    try:
+        cpu_prog.compile(tmpdir=temp_dir)
+    finally:
+        shutil.rmtree(temp_dir)
+    saved_sys_path = sys.path
+    try:
+        sys.path.append(temp_dir)
+        cpu_func = getattr(importlib.import_module("_cpu_obj.lib"), func_name)
```

Comment:
It's great that there exists a `tmpdir` argument 👍 But I am very confused: We build an extension module, and then remove it (line 82) before loading the module? How does it work?

Reply:
I can only plead temporary insanity :) Fixed.